### DISCUSSION TITLE
Battery Status extension request for laptop users

### DISCUSSION THREAD

**Discussion by SandipanGuha** [brackets-cont/brackets#303] [Feature Request]
I work long hours on my laptop and sometimes I forget to check my battery status.
It would be really helpful if I could get the battery status indicator right in my Brackets statusbar.
It should only show the current battery percentage along with the charging status. If it could provide a notification popup for "fully charged" and 
"low battery", would be more than my expectation.
Thanks for this wonderful IDE. 
❤️ from EnblazE

**Comment by ScanuNicco** [comment]
This should be relatively easy to implement. It might even be possible to make it work on the online version of phcode on Chromium based browsers. Putting this here for reference:
https://developer.mozilla.org/en-US/docs/Web/API/Battery_Status_API
